### ISSUE TITLE
Integrated Security Mode = 2 and receiving 401 authentication error.

### ISSUE THREAD

**Issue by austin98jr** [cubewise-code/tm1py#1119]
I am trying to connect to my TM1 server via TM1py and am receiving the following error:

Text: '' - Status Code: 401 - Reason: 'Unauthorized' - Headers: {'Content-Type': 'text/plain', 'Content-Length': '0', 'Connection': 'keep-alive', 'Set-Cookie': 'TM1SessionId=hhRW6fBgZeeTEwf-8WwB5A; Path=/api/; HttpOnly; Secure', 'WWW-Authenticate': 'Negotiate, Basic realm="TM1"'}

I can access cubes on this server via architect using this account.

IntegratedSecurityMode=2 
SecurityPackageName=KERBEROS
HTTPPortNumber=5556
Here is the code that I am using:

``` python
import logging
from TM1py import TM1Service
from TM1py.Exceptions import TM1pyException
from TM1py.Objects import NativeView, MDXView
import pandas as pd


# Main script
def main():
    logging.basicConfig(level=logging.DEBUG)
    with TM1Service(address='localhost', port=5556, ssl=True, integrated_login=True) as tm1:


        tm1.logout()
    

# Run the main script
if __name__ == "__main__":
    main()
```

**Comment by MariusWirtz** [comment]
Please check if this resolves the issue:
https://github.com/cubewise-code/tm1py/issues/1065#issuecomment-2042465896

**Comment by austin98jr** [comment]
> Please check if this resolves the issue: [#1065 (comment)](https://github.com/cubewise-code/tm1py/issues/1065#issuecomment-2042465896)

Solved it! Thank you!

**Comment by austin98jr** [comment]
> Please check if this resolves the issue: [#1065 (comment)](https://github.com/cubewise-code/tm1py/issues/1065#issuecomment-2042465896)

Solved it thank you!
